code/src/components/Header.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+
+export const Header = () => {
+  return (
+    <nav className="nav-links">
+      <div className="nav-link-item">
+        <NavLink className="nav-bar-link" exact to={"/"}>
+          Home
+        </NavLink>
+      </div>
+      <div className="nav-link-item">
+        <NavLink className="nav-bar-link" exact to={"/contactUs"}>
+          Contact Us
+        </NavLink>
+      </div>
+    </nav>
+  );
+};
```

Comment:
Looks really good that you added a header, and clever to add the contact information in a link!